platform-sdk/swirlds-logging-log4j-appender/src/test/java/com/swirlds/logging/log4j/appender/SwirldsLogAppenderTest.java

```diff
@@ -0,0 +1,191 @@
+/*
+ * Copyright (C) 2024 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.swirlds.logging.log4j.appender;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import com.swirlds.config.api.Configuration;
+import com.swirlds.config.extensions.test.fixtures.TestConfigBuilder;
+import com.swirlds.logging.api.Level;
+import com.swirlds.logging.api.internal.LoggingSystem;
+import com.swirlds.logging.legacy.LogMarker;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Stream;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.apache.logging.log4j.Marker;
+import org.apache.logging.log4j.MarkerManager;
+import org.apache.logging.log4j.ThreadContext;
+import org.apache.logging.log4j.core.LoggerContext;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+class SwirldsLogAppenderTest {
+    private static final String LOGGER_NAME = "testLogger";
+
+    public LoggingSystem setup(final com.swirlds.config.api.Configuration configuration) {
```

Comment:
that is a factory instead of a set. Better call it "createLoggingSystem", move it to the bottom of the file and make it static